sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -325,131 +337,161 @@ static int get_operation_status()
   char* topic;
   int topic_len;
   MQTTClient_message* message;
+  az_iot_provisioning_client_register_response response;
+  az_iot_provisioning_client_operation_status operation_status;
 
   // Continue to parse incoming responses from the Provisioning Service until
   // the device has been successfully provisioned or an error occurs
   while (!is_operation_complete)
   {
+    // Receive message.
+    // If MQTTClient_receive is successful, topic and message memory must later be freed
```

Comment:
End sentence with "."

Reply:
I'm seeing almost all comments not have a . at the end in many files.  Also with regards to printfs, some have . and some don't.  I think something to bring up at meeting monday on the standard to go with.

Reply:
I noticed that the issue is localized to our samples: all other files I've looked at have the proper English syntax period at end of sentence. Please do fix all places that you've found.

Reply:
ah ok.  I will do a sweep on all sample files in another task so I don't forget any.  Easier to track.